File: java/URI_1065.java
package URI;
import java.util.Scanner;

public class URI_1065 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int totpar = 0;
		
		for(int i = 0; i < 5; i++) {
			int n = sc.nextInt();
			if(n % 2 == 0) {
				totpar += 1;
			}			
		}
		System.out.println(totpar + " valores pares");
		
		sc.close();

	}

}

File: kotlin/URI_1065.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var pares = 0
    for (i in 1..5) {
        val n = sc.nextInt()
        if (n % 2 == 0) pares++
    }
    println("$pares valores pares")
}
